### PR TITLE
Remove broken readability link

### PR DESCRIPTION
**Pull request by budmc29**
The better option would be to replace the readability link with the correct one, but I don't know which one is that.

### CODE REVIEW

File: best-practices/README.md [hunk 1]
```diff
@@ -12,9 +12,7 @@ General
 * Don't swallow exceptions or "fail silently."
 * Don't write code that guesses at future functionality.
 * Exceptions should be exceptional.
-* [Keep the code simple].
-
-[Keep the code simple]: http://www.readability.com/~/ko2aqda2
+* Keep the code simple.
```
Comment (bf4) [inline best-practices/README.md]: wow, looks like those have been there forever https://github.com/thoughtbot/guides/commit/04b6e2ea8597ae46b24247b96da9b04645478fa0

Comment (iwz) [inline best-practices/README.md]: @jferris any idea where this link went?

Comment (croaky) [inline best-practices/README.md]: I wonder if we linked to the Rich Hickey "Sinple Made Easy" talk to describe "simplicity" in code?

https://github.com/matthiasn/talk-transcripts/blob/master/Hickey_Rich/SimpleMadeEasy.md

Cool transcript version ^

**Comment by croaky** [comment]
Thanks @budmc29! Merged as 752db69

**Pull request closed without merging**